diff --git a/index.js b/index.js
index b6511d0e0fc3c814e7f294e31699a7fae3650de5..a3fcb0e7e38b3a7386c202867ba3303e7a7bcf36 100644
--- a/index.js
+++ b/index.js
@@ -97,18 +97,21 @@ module.exports._getDependencies = function(config = {}) {
   const resolvedDependencies = [];
 
   for (const dependency of dependencies) {
-    const result = cabinet({
-      partial: dependency,
-      filename: config.filename,
-      directory: config.directory,
-      ast: precinct.ast,
-      config: config.requireConfig,
-      webpackConfig: config.webpackConfig,
-      nodeModulesConfig: config.nodeModulesConfig,
-      tsConfig: config.tsConfig,
-      tsConfigPath: config.tsConfigPath,
-      noTypeDefinitions: config.noTypeDefinitions
-    });
+    let result;
+    try {
+      result = cabinet({
+        partial: dependency,
+        filename: config.filename,
+        directory: config.directory,
+        ast: precinct.ast,
+        config: config.requireConfig,
+        webpackConfig: config.webpackConfig,
+        nodeModulesConfig: config.nodeModulesConfig,
+        tsConfig: config.tsConfig,
+        tsConfigPath: config.tsConfigPath,
+        noTypeDefinitions: config.noTypeDefinitions
+      });
+    } catch {}
 
     if (!result) {
       debug(`skipping an empty filepath resolution for partial: ${dependency}`);
